google/cloud/spanner/transaction.h

```diff
@@ -0,0 +1,156 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_SPANNER_TRANSACTION_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_SPANNER_TRANSACTION_H_
+
+#include "google/cloud/spanner/timestamp.h"
+#include "google/cloud/spanner/version.h"
+#include <google/spanner/v1/transaction.pb.h>
+#include <chrono>
+#include <memory>
+
+namespace google {
+namespace cloud {
+namespace spanner {
+inline namespace SPANNER_CLIENT_NS {
+
+namespace internal {
+class Transaction;
+}
+
+/**
+ * A transaction is a set of reads and writes that execute atomically at a
```

Comment:
This is a bit long for the "brief" description... maybe: "A wrapper for Cloud Spanner transactions. Ties read and write operations together."

Reply:
Done.